tpcx_bb/queries/q18/tpcx_bb_query_18.py

```diff
@@ -281,7 +281,8 @@ def main(client, config):
 
     # This file comes from the official TPCx-BB kit
     # We extracted it from bigbenchqueriesmr.jar
-    with open("negativeSentiment.txt") as fh:
+    sentiment_dir = "/".join(config["data_dir"].split("/")[:-3] + ["sentiment_files"])
```

Comment:
This could probably be a utility, since the structure is explicit and it's used six times across queries. In a followup PR.